lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -263,6 +263,14 @@
     - ' - C(type) (string): Value type, string type by default.'
     - ' - C(operation): C(remove): This attribute is required only when removing properties'
     version_added: '2.6'
+  datastore:
+    description:
+    - Specify datastore or datastore cluster to provision virtual machine.
+    - 'This will take precendence over "disk.datastore" parameter.'
```

Comment:
Does this mean that setting the datastore for each disk separately will no longer work?

Reply:
Currently, specifying separate datastore in disk is not working either.

https://github.com/ansible/ansible/blob/87d6bdaf9819fdc45a5d309e5e54f8b6f6651ada/lib/ansible/modules/cloud/vmware/vmware_guest.py#L1442